### ISSUE TITLE
Notification System for New Orders

### ISSUE THREAD

**Issue by aknakshay** [shubh28/safeslot#64]
After https://github.com/shubh28/safeslot/issues/18

Whenever a new order is created, we should give a notification to a store owner with a sound, that there's a new order.

**Comment by christenmartin** [comment]
Would this be a good use case for socket.io?

**Comment by shubham-jain4** [comment]
@christenmartin yes good use case for sockets but will need to integrate whole socket implementation. We can think of long polling as well here
